lib/bolt/util.rb

```diff
@@ -0,0 +1,47 @@
+module Bolt
+  module Util
+    # CODEREVIEW I hate mixing in random modules
+    class << self
+      def read_config_file(path, default_paths = nil)
```

Comment:
Extracting these makes some sense. This version isn't being used universally though, which it should.